rpcd/playbooks/roles/rpc_maas/library/raxmon.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python
+
+# Copyright 2017, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import ConfigParser
+
+from ansible.module_utils.basic import AnsibleModule
+
+
+def _get_agent_tokens(conn, entity):
+    agent_tokens = []
+    for a in conn.list_agent_tokens():
+        if a.label == entity:
+            agent_tokens.append(a)
+    return agent_tokens
+
+
+def _get_conn(get_driver, Provider):
+    cfg = ConfigParser.RawConfigParser()
+    cfg.read('/root/.raxrc')
+    driver = get_driver(Provider.RACKSPACE)
+    conn = None
+    try:
+        user = cfg.get('credentials', 'username')
+        api_key = cfg.get('credentials', 'api_key')
+        conn = driver(user, api_key)
+    except (ConfigParser.NoSectionError, ConfigParser.NoOptionError):
+        url = cfg.get('api', 'url')
+        token = cfg.get('api', 'token')
+        conn = driver(None, None, ex_force_base_url=url,
+                      ex_force_auth_token=token)
```

Comment:
Yeah, this is a bit messy.  TBH, this was ripped from rpc-maas-tool.py, and perhaps what we'd be better off doing is putting this code into rackspace-monitoring or rackspace-monitoring-cli libraries so that every tool we write doesn't have to rewrite this config file handling.  Would that be fair?